molecule/in/requirements.yml

```diff
@@ -0,0 +1,7 @@
+---
+# PostgreSQL
+- name: postgresql
+  src: ANXS.postgresql
```

Comment:
I haven't done it yet in my older codebase yet but we might consider using the [geerlingguy postgresql role](https://github.com/geerlingguy/ansible-role-postgresql) as it is maintained a bit better than this one.

Reply:
We may have to move to that in the future, but for the test suite it still works.